demisto_sdk/commands/lint/tests/test_linter/docker_runner_test.py

```diff
@@ -32,13 +33,15 @@ def test_build_image_no_errors(self, linter_obj: Linter, mocker):
         mocker.patch.object(linter_obj, '_docker_client')
         docker_build_response = mocker.MagicMock()
         docker_build_response.short_id = exp_test_image_id
-        linter_obj._docker_client.containers.create().commit().short_id = exp_test_image_id
+
+        linter_obj._docker_client.images.build().__getitem__().short_id = exp_test_image_id
 
         act_test_image_id, act_errors = linter_obj._docker_image_create(docker_base_image=[exp_test_image_id, 3.7])
 
         assert act_test_image_id == exp_test_image_id
         assert act_errors == exp_errors
-        assert linter_obj._docker_client.images.build.call_count == 0
+        # Due to some mocker issues
+        assert linter_obj._docker_client.images.build().__getitem__.call_count == 2
```

Comment:
1. Are we decided to continue with this? @reutshal 
2. If yes, I think it's better to give a better description in the comment, So if bug occurred the developer will not need to search the reason for it.

Reply:
Actually I think we can remove this assertion.
Not sure how relevant it is anymore